pom.xml

```diff
@@ -61,6 +61,19 @@
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+        </dependency>
```

Comment:
Do you really need both annotations? you already have `spring-boot-starter-validation`